manager/manifests/istio.yaml

```diff
@@ -47,18 +47,6 @@ gateways:
     - port: 8060
       targetPort: 8060
       name: tcp-citadel-grpc-tls
-    - port: 15029
-      targetPort: 15029
-      name: http2-kiali
-    - port: 15030
-      targetPort: 15030
-      name: http2-prometheus
-    - port: 15031
-      targetPort: 15031
-      name: http2-grafana
-    - port: 15032
-      targetPort: 15032
-      name: http2-tracing
     tracing:
```

Comment:
Is any more of the config able to be deleted so that defaults are used? Or is this the minimum?